packages/Algernon.package/ALGWorldMenuMorphType.class/README.md

```diff
@@ -1 +1 @@
-An ALGWorldMenuMorphType represents a Morph that appears in the world menu 'open' menu. The typeObject for this ALGType should be in the same format as that in TheWorldMenu registry.
\ No newline at end of file
+An ALGTestType represents a test in the system.
```

Comment:
Why did you replace this comment? That's the wrong one.

Reply:
was an unwanted change, probably in the mergin process, is now changed back